registry/service/service.go

```diff
@@ -58,8 +58,13 @@ func (s *serviceRegistry) Register(srv *registry.Service, opts ...registry.Regis
 		o(&options)
 	}
 
+	ctx := context.Background()
+	if options.TTL.Nanoseconds() != 0.0 {
+		ctx = context.WithValue(ctx, "register_ttl", options.TTL)
```

Comment:
This won't actually do anything, We need to either pass TTL in the proto as part of some sort of new options field or ignore entirely and assume a default TTL for the registry service.

Reply:
This can probably be removed

Reply:
Actually. This should be updated to send TTL in the options right?

Reply:
Yeah, good spot! Will push the change shortly.